dist/components/tangojs-image.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html>
+<html>
+
+<script type="text/javascript">
+(function (window) {
+  'use strict'
+
+  const document = window.document
+  const tangojs = window.tangojs
+
+  
+  class TangoJsImageElement extends window.HTMLElement {
+
+    createdCallback () {
+      console.log('createdCallback')
+
+      this.root = this.createShadowRoot()
+      this.dom = this.createLocalDom()
+      this.root.appendChild(this.dom.grid)
+
+      //TODO instancier composant ici
+      this.viewer = this.createViewer(this.dom.grid)
+
+      this.onResizeCallback = () => {
+
+      }
+
+      this.mutationObserver = new MutationObserver(this.onResizeCallback)
```

Comment:
Maybe you don't need a MutationObserver here. Can attributeChangedCallback do the job here?
(see: https://developer.mozilla.org/en-US/docs/Web/Web_Components/Custom_Elements)

If you just want to listen for changes in attribute values, you can also use `mixins.withReflectedAttributes` - it automagically maps HTML attributes to javascript properties of `this` (the widget) and takes care of conversions to/from DOM strings.